appveyor.yml

```diff
@@ -1,60 +1,65 @@
-image: Visual Studio 2019
+# Source: https://github.com/krlmlr/r-appveyor/blob/master/appveyor.yml
 
-version: '{build}'
+# DO NOT CHANGE the "init" and "install" sections below
 
 # Download script file from GitHub
 init:
-  - ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"
-  - ps: |
-      $ErrorActionPreference = "Stop"
-      Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
-      Import-Module '..\appveyor-tool.ps1'
+  ps: |
+        Get-Date
+        $ErrorActionPreference = "Stop"
 
 install:
-  - ps: Bootstrap
-  - git submodule update --init --recursive
```

Comment:
No submodules present in this repo

Reply:
I am very confused here. Why do we have appveyor in this repo? I thought everything was handled using a different strategy because it was on CRAN?

Reply:
Please keep the bootstrap (this is required to build)
The git submodule can be removed, but not because we do not have submodule. It is done for all package on appveyor by default